### DISCUSSION TITLE
Тутор по запуску

### DISCUSSION THREAD

**Discussion by Semeneater** [lirix360/ReadmangaGrabber#29] [General]
Здравствуйте, можете запилить какой-нибудь тутор в скринах или видео как запустить программу?
Далек от программирования не выкупаю что делать с файлами из ZIP.

**Comment by Semeneater** [comment]
Ладно, я сам разобрался. 
1) Скачать Go: golang.org
2) Распаковать ZIP
3) В папке открыть PowerShell
4) Ввести go get -u -v -f all
5) Ввести go run main.go

**Reply by lirix360** [reply]
Или просто скачать архив с уже собранной версией из раздела релизов.

**Reply by Hanqnero** [reply]
Здравствуйте, на самом деле всё проще. Вы скачали исходный код проекта, компилятор и запустили программу его средствами.
Вы могли скачать уже готовый исполняемый файл во вкладке с релизами, скачать релиз для Windows и запуспить файл .exe после распаковки архива.
![grabber](https://user-images.githubusercontent.com/57258124/145607382-0f31f6f3-2d21-45f2-a548-1fa584c58a79.png)

**Reply by Semeneater** [reply]
Cпасибо!

**Reply by RarogCmex** [reply]
Справедливости ради, инструкцией @Semeneater можно запустить под Linux/Mac

**Reply by SiriusKyoko** [reply]
> Здравствуйте, на самом деле всё проще. Вы скачали исходный код проекта, компилятор и запустили программу его средствами. Вы могли скачать уже готовый исполняемый файл во вкладке с релизами, скачать релиз для Windows и запуспить файл .exe после распаковки архива. ![grabber](https://user-images.githubusercontent.com/57258124/145607382-0f31f6f3-2d21-45f2-a548-1fa584c58a79.png)

Я так и не поняла как здесь мангу качать... Более-менее разобралась как найти этот файл, скачала вроде, открыть не могу...

**Comment by nik-nantes** [comment]
Открываю в браузере запуская index, подставляю ссылку с минтманги но команды список глав и загрузка не работают, просто бесконечное колесо загрузки. Пробовал в 3 разных браузерах: yandex, edge и opera.

**Reply by lirix360** [reply]
1) В смысле "запуская index"? Вы скачиваете исходники и открываете index.html? Так работать не будет, нужно скачивать бинарник из радела [релизов](https://github.com/lirix360/ReadmangaGrabber/releases) или компилировать скачанные исходники.
2) Минтманга реализована в эксперементальном режиме и может не работать.
